ethereum/contracts/pyth/Pyth.sol

```diff
@@ -68,8 +68,8 @@ abstract contract Pyth is PythGetters, PythSetters, AbstractPyth {
         emit UpdatePriceFeeds(msg.sender, updateData.length, requiredFee);
     }
 
-    function getUpdateFee(uint updateDataSize) public override view returns (uint feeAmount) {
-        return singleUpdateFeeInWei() * updateDataSize;
+    function getUpdateFee(bytes[] calldata updateData) public override view returns (uint feeAmount) {
```

Comment:
Isn't this a breaking API change? How are you planning to do the migration?

Reply:
evm != aptos ser. Evm doesn't have upgrades native to have the notion of breaking at all. So in theory nothing is breaking and you can even break the storage if you want to.

Calling a non-existing method results in a revert without any message and that could be a problem if there were protocols already live on top of us, but as there are none, this is fine.

Reply:
Yeah, I was concerned about breaking protocols that are already live using us.